datasets/swiss_judgment_prediction/swiss_judgment_prediction.py

```diff
@@ -119,56 +136,62 @@ def _split_generators(self, dl_manager):
             dl_dir = dl_manager.download_and_extract(urls_to_dl)
```

Comment:
I just realized there is no need of extracting because the files are not compressed:
```suggestion
            dl_dir = dl_manager.download(urls_to_dl)
```

Also you could remove the line:
```python
urls_to_dl = _URLS
```
and replace the downloading line with:
```python
dl_dir = dl_manager.download(_URLS)
```

Reply:
thanks, I updated the code with this.